Code/Framework/AzCore/Platform/Android/AzCore/std/parallel/internal/thread_Android.cpp

```diff
@@ -25,11 +25,9 @@ namespace AZStd
             // Unimplemented on Android
         }
 
-        void SetThreadPriority(int priority, pthread_attr_t&)
+        void SetThreadPriority([[maybe_unused]] int priority, pthread_attr_t&)
         {
             // (not supported at v r10d)
-            AZ_Warning("System", priority <= 0, "Thread priorities %d not supported on Android!\n", priority);
-            (void)priority;
```

Comment:
Removed this spamming message, which doesn't add value at the moment. It'd be better to create a github task instead.

Reply:
This shouldn't have been a warning in the first place. If anything, there should be a PAL 'isThreadPrioritySupported()' function in conjunction with this call instead